### DISCUSSION TITLE
Do i need to learn JS before solidity?

### DISCUSSION THREAD

**Discussion by naitikshrma** [Cyfrin/foundry-full-course-cu#763] [General]
Do i need to learn JS before solidity?

**Comment by theirrationalone** [comment]
You must at least have Basic `Javascript` Knowledge Therefore `JS` is the monopolist in the web tech world.

**Reply by naitikshrma** [reply]
Can I do it simultaneously?

**Reply by theirrationalone** [reply]
Why not... You can do absolutely... 

Or you can join [alchemy university 🎓](https://university.alchemy.com/), where they introduce js to you and full web3. 

Or Patrick also has a web3 course based on javascript... [Patrick web3 HH](https://youtu.be/gyMwXuJrbJQ?si=U8KAMNKtRupE5kIc)

**Comment by drained69** [comment]
You do. It would make it easier for you.

**Reply by naitikshrma** [reply]
sure ...thanks
